pkg/process/manager.go

```diff
@@ -180,6 +167,16 @@ func (m *Manager) buildStartCommand(d *daemon.Daemon) (*exec.Cmd, error) {
 		}
 	}
 
+	args = append(args, "--apisock", d.GetAPISock())
+	args = append(args, "--log-level", d.LogLevel)
+	if !d.LogToStdout {
+		args = append(args, "--log-file", d.LogFile())
+	}
+	nydusdThreadNum := d.NydusdThreadNum()
+	if nydusdThreadNum != "" {
+		args = append(args, "--thread-num", nydusdThreadNum)
```

Comment:
`--thread-num` only works for FUSE daemon?

Reply:
sorry, it should be "--fscache-threads" for fscache, fixed.